app/models/supported_ssg.rb

```diff
@@ -18,6 +18,12 @@ def comparable_version
     Gem::Version.new(version)
   end
 
+  def version_with_revision
+    Gem::Version.new(
+      package.sub(/^scap-security-guide-(.*)\.el.*$/, '\1')
```

Comment:
Observation: for an unmatched package pattern this would raise `Malformed version number string ... (ArgumentError)`. That might or might not be deliberate.

Reply:
I think we want this to fail then, smoke tests would catch it.

Reply:
Smoke tests would, however we get the data from external source (compliance-ssg).

Reply:
Correction: we'd catch this in stage :laughing: and we rather want to